GitCommands/Git/GitModule.cs

```diff
@@ -389,6 +389,47 @@ public static bool IsValidGitWorkingDir(string dir)
                    Directory.Exists(dirPath + "refs");
         }
 
+        /// <summary>
+        /// Asks git to resolve the given relativePath
+        /// git special folders are located in different directories depending on the kind of repo: submodule, worktree, main
+        /// See https://git-scm.com/docs/git-rev-parse#git-rev-parse---git-pathltpathgt
+        /// </summary>
+        /// <param name="relativePath"></param>
+        /// <returns></returns>
+        public string GetGitPath(string relativePath)
```

Comment:
`GetGitPath` name does not suggest that _asks git to resolve the given relativePath_. It more looks like it returns the path where git is.
Perhaps rename it to something like `ResolvePath`. 
There is no need to use `Git` in method name since it is located in `GitModule` class of `GitCommand` namespace